src/main/java/com/jcabi/dynamodb/maven/plugin/AbstractDynamoMojo.java

```diff
@@ -120,11 +121,20 @@ protected int tcpPort() {
     }
 
     /**
+     * Set the project environment.
+     * {@link com.jcabi.dynamodb.maven.plugin.AbstractEnviromentMojo}.
+     * @throws MojoFailureException If fails
+     * @checkstyle Intentionally empty!, To be implemented by sub classes!
```

Comment:
@simonjenga why you need `@checkstyle` annotation here ?
please remove the annotation and move the comment up to the second line